README.md

```diff
@@ -77,6 +78,12 @@ If you are using a secured instance of SonarQube, you can provide a SonarQube au
 java -jar cnesreport.jar -t xuixg5hub345xbefu -s https://example.org:9000 -p projectId -r ./template.docx
 ````
 
+##### Export of a specific project branch (standalone)
+If you are using a commercial edition of sonarqube or the [sonarqube-community-branch-plugin](https://github.com/mc1arke/sonarqube-community-branch-plugin) you want to export the report for a specific branch of your project using the -b option. 
```

Comment:
Could you please replace " **you want to** export the report" by "**, you can** export the report" ?
Just in order to match the sentence format of the other sections.

Reply:
```suggestion
If you are using a commercial edition of sonarqube or the [sonarqube-community-branch-plugin](https://github.com/mc1arke/sonarqube-community-branch-plugin) you can export the report for a specific branch of your project using the -b option. 
```